osgi/adapter-tests/osgi-adapter-test/src/test/java/org/jvnet/hk2/osgiadapter/ServiceLocatorHk2MainTest.java

```diff
@@ -157,6 +155,10 @@
 		        provision(mavenBundle().groupId(GROUP_ID).artifactId(
 		                        "sdp-management-bundle").version(projectVersion).startLevel(4)),
                                 provision(mavenBundle().groupId("javax.el").artifactId("javax.el-api").version("2.2.5")),
+                                provision(mavenBundle().groupId("javax.validation").artifactId("validation-api").version("1.1.0.Final")),
+                provision(mavenBundle().groupId("org.hibernate").artifactId("hibernate-validator").version("5.1.2.Final")),
+                provision(mavenBundle().groupId("com.fasterxml").artifactId("classmate").version("1.0.0")),
+                provision(mavenBundle().groupId("org.jboss.logging").artifactId("jboss-logging").version("3.1.3.GA")),
```

Comment:
can you upgrade to jboss-logging 3.3.0.Final also? It's the properly OSGied version.